### ISSUE TITLE
value of the loss function

### ISSUE THREAD

**Issue by BartoszKosciug** [satojkovic/DeepLogo#31]
What was the the value of the total loss function at training process termination, in your case?

**Comment by satojkovic** [comment]
Hi @TacoTijuana 

When I evaluated the trained model using eval.py, I got the loss values below.

| classification loss | localization loss |
|:-:|:-:|
| 2.721 @ 6.046k step | 0.44 @ 6.046k step | 

Also mAP result is the following. 

```bash
PascalBoxes_Precision/mAP@0.5IOU: 0.872595
```

**Comment by yumiaoGitHub** [comment]
would you share with us the "eval.py"? That would be helpful, Thanks!

**Comment by satojkovic** [comment]
hi @yumiaoGitHub 
 
Here it is.
https://github.com/tensorflow/models/blob/master/research/object_detection/legacy/eval.py
